x-pack/plugins/synthetics/server/synthetics_service/service_api_client.ts

```diff
@@ -225,23 +242,38 @@ export class ServiceAPIClient {
 
     const authHeader = this.authorization ? { Authorization: this.authorization } : undefined;
 
+    const data = this.getRequestData(serviceData);
+
+    if (method === 'INSPECT') {
+      return data;
```

Comment:
`INSPECT` calls shouldn't be a method to `callServiceEndpoint` as service isn't being called and thus `Successfully called service location ...` and `Failed to call service location  ...` shouldn't be logged. Doing this also changes the return type of `callServiceEndpoint`.